flow-editor/src/visual-node-editor/VisualNodeEditor.tsx

```diff
@@ -162,6 +164,7 @@ export type ClipboardData = {
 export type GroupEditorBoardData = {
   viewPort: ViewPort;
   selected: string[];
```

Comment:
now that we have selected connections "selected" becomes (even more) vague
```suggestion
  selectedInstances: string[];
```